### PR TITLE
Release 2.3.1, with name change to CaratInterface

### PR DESCRIPTION
**Pull request by gaehler**
Renamed Carat to CaratInterface.

### CODE REVIEW

**Comment by gaehler** [comment]
Hmm, CARAT is compiled successfully, but CaratInterface is not loaded. Locally, on my system, it does work. Again a parallel make issue, @fingolfin ?

**Comment by fingolfin** [comment]
Hm, will have to look... The `Makefile` indeed isn't quite safe for parallelization, but I don't see an immediate problem in the Travis logs. I do wonder whether the right version of the package gets loaded, though, hm.

BTW, I just realized one concern about renaming the package, by the way (CC @alex-konovalov): There isn't really way to express this to the package load system. So if somebody for some reason manages to have both `carat` and `CaratInterface` in their GAP `pkg` directory, then they could try to load both, which of course would result in a terrible mess and hopefully lots of errors. Alas, I think that's probably OK...

The other thing is that (I believe) `carat` was on the list of autoloaded packages, while `CaratInterface` is not.

None of this is vital, I think, but I wanted to mention it anyway, just so you are fully informed.

**Comment by gaehler** [comment]
Loading the package explicitly helped, only codecov is not yet happy. Also, I haven't adjusted the links to travis and codecov in README.md yet. Is there any setup to be changed before I do that,
@alex-konovalov? It seems to me that there is nothing serious left, so that I could merge this and publish an actual release, which then can be tested with the rest of GAP, perhaps even with GAP on Windows. If there is still significant further development in CARAT soon, I can still make another release.

File: tst/testall.g [hunk 1]
```diff
@@ -1,3 +1,4 @@
+LoadPackage( "CaratInterface" );
```
Comment (fingolfin) [inline tst/testall.g]: Ahh, of course, I should have realized this was missing. I even looked at this file. Sorry

**Comment by alex-konovalov** [comment]
Thanks @gaehler - I suggest to adjust links to travis and codecov in README.md and go ahead with the release. I will take care of package update system, and on not including both old and new package in the merged package archives / Docker containers etc.

**Comment by fingolfin** [comment]
Ignore the codecov "complaint", it just says that the code you touch is only covered 66%, which is below average for the project, but this is really meaningless here.

One thing that I find really confusing is why this project has both a README and a README.md. I'd recommend to delete one. That means less work for you and less confusion for users. 


Other than that, I think there is nothing else to be done on your end, go ahead and make that release :-).

**Comment by fingolfin** [comment]
As @alex-konovalov said, you should update the travis/codecov links in `README.md`. While at it, one remove some superfluous bits in them, too, and arrive at this:
```
[![Build Status](https://travis-ci.org/gap-packages/CaratInterface.svg)](https://travis-ci.org/gap-packages/CaratInterface)
[![Code Coverage](https://codecov.io/github/gap-packages/CaratInterface/coverage.svg)](https://codecov.io/gh/gap-packages/CaratInterface)
```

**Pull request merged**